app/uk/gov/hmrc/customs/declaration/model/batchFileUploadMetadataModels.scala

```diff
@@ -73,7 +73,7 @@ case class BatchFile(
   location: URL, // upscan-initiate
   sequenceNumber: FileSequenceNo, // derived from user request
   size: Int, // assumption - it appears to be mandatory but is ignored
```

Comment:
how about `maybeDocumentType` which seems to be our convention

Reply:
Skipped this as it alters persisted json